src/index.js

```diff
@@ -35,8 +47,31 @@ class GeoRaster {
     if (debug) console.log('this after construction:', this);
   }
 
+  async getValues(left, top, right, bottom, width, height) {
```

Comment:
looks great so far! Is it possible to make some of these values optional?

Reply:
I'm going to resolve this.  Would be nice to make some of the values optional in a subsequent PR, but not necessary before merging this one.